src/ctia/stores/es/sort.clj

```diff
@@ -18,6 +29,35 @@
       ;; https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_sort_values
       {field-name {:order order}}
 
+      ;; chooses the maximum value in field (a list) based on remappings,
+      ;; and sorts based on that value for each entity. Remappings must
+      ;; be positive integers.
+      ;; eg
+      ;; {:op :remap-list-max
+      ;;  :field-name "tactics"
+      ;;  :remappings {"TA0043" 1
+      ;;               "TA0042" 2}
+      ;;  :sort_order :asc}
+      :remap-list-max
+      (let [{:keys [remap-default remappings]} params
+            remappings (normalize-remappings remappings)]
+        (assert (every? (complement neg?) (vals remappings)) ":remap-list-max :remappings values must be non-negative")
+        {:_script
+         {:type "number"
+          :script {:lang "painless"
```

Comment:
painless script isn't available for ES5, is it?

Reply:
It is https://www.elastic.co/blog/painless-a-new-scripting-language